api/src/main/java/org/openmrs/module/attachments/AttachmentsContext.java

```diff
@@ -345,6 +345,12 @@ public static double getCompressionRatio(double fileByteSize, double maxByteSize
 	 */
 	public static ContentFamily getContentFamily(String mimeType) {
 		ContentFamily contentFamily = ContentFamily.OTHER;
+		if (StringUtils.equals(mimeType, "some type")) {
+			contentFamily = ContentFamily.EXECUTABLE;
+		}
+		if (StringUtils.equals(mimeType, "some type")) {
```

Comment:
Unfortunately i haven't been able to get the correct MIME type for all scripts.

Reply:
why not simply check the type you wana exclude here and return the appropriate message openmrs-module-attachments/AttachmentResource1_10.java at bd8f77ba063203a6898b33ef4a856c64444b9738 · openmrs/openmrs-module-attachments · GitHub